docs/src/main/asciidoc/rest-client-reactive.adoc

```diff
@@ -238,6 +238,15 @@ quarkus.rest-client.extensions-api.url=https://stage.code.quarkus.io/api
 quarkus.rest-client.extensions-api.scope=javax.inject.Singleton
 ----
 
+=== Disabling Hostname Verification
+
+To disable the SSL hostname verification for a specific REST client, add the following property to your configuration:
+
+[source,properties]
+----
+quarkus.rest-client.extensions-api.verify-host=false
```

Comment:
Asking that because somebody asked with the behavior of `quarkus.tls.trust-all` in conjunction with REST Clients. That being said, being able to adjust things specifically for REST Client would be useful anyway so I think this patch has value.

Reply:
Yes, `quarkus.tls.trust-all` is global. Having a rest client attribute would be nice. However, I would keep the name `trust-all`.

Reply:
Are you talking about a (per-client) `trust-all` that skips more verifications _instead_ of `verify- host` that skips only the hostname verification?
Or is this just a naming discussion?

Reply:
Trust-all by-pass everything (basically pass a ssl context that accept any certs).

Reply:
Yes, that's why I'm asking. `trust-all` (global or per client) is a sledgehammer which is usually unacceptable in production (where you might have cases though in which you can only use an IP to talk to another system).
So my point is that `trust-all` (on whichever level) should not "replace" `verify-host`.

So, on client level, `trust-all` _additionally_ to `verify-host` would be ok for me, but having _only_ `trust-all` is much too coarse.

Reply:
I agree with @famod on this.